src/components/closeModalCountdown.js

```diff
@@ -0,0 +1,16 @@
+import React, { useEffect, useState } from 'react'
+import { ReminderButton } from './buttons'
+
+export default function CloseModalCountdown({ closeModal }) {
+  const [counter, setCounter] = useState(3)
```

Comment:
This looks like it is foremost a custom hook, maybe extract it and use the hook in the parent component.